packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -155,13 +153,10 @@ const BasicCell: React.FC<TableCellProps> = ({ sx, ...props }) => (
 );
 
 export const BatchesTable: React.FC<BatchesTableProps> = ({
-  item,
   onChange,
   register,
   rows,
 }) => {
-  if (!item) return null;
-
```

Comment:
this was all it was used for, and now there's an `isLoading` state for fetching the batches which i'm using to conditionally render in the parent